test/pummel/test-http-upload-timeout.js

```diff
@@ -57,7 +60,8 @@ server.listen(0, function() {
 
       function ping() {
         const nextPing = (Math.random() * 900).toFixed();
-        if (nextPing > 600) {
+        // Do not set too big to prevent req.socket from emitting timeout
+        if (nextPing > 300) {
```

Comment:
Wouldn't it be better to use `(Math.random() * 301).toFixed()` and remove the `if` condition?

Reply:
I think `(Math.random() * 301).toFixed()` is ok. But without `if`,how do we decide when to call end?

Reply:
Ah, right. I'm fine with keeping it as is then.